lib/publiccloud/instance.pm

```diff
@@ -487,4 +513,176 @@ sub network_speed_test() {
     record_info("curl $rmt_host", $self->run_ssh_command(cmd => "curl -w '$write_out' -o /dev/null -v https://$rmt_host/", proceed_on_failure => 1));
 }
 
+=head2 measure_boottime
+
+    measure_boottime();
+
+Perfomrance measurement of the system Boot time. 
+Mainly used C<systemd-analyze> command for the data extraction.
+Data is then collected in an internal record, ready for storing in a DB.
+Set PUBLIC_CLOUD_PERF_COLLECT true or >0, to activate boottime measurements.
+
+=cut
+
+sub measure_boottime() {
+    my ($self, $instance, $type) = @_;
+    return 0 unless (get_var('PUBLIC_CLOUD_PERF_COLLECT'));
+
+    my $ret = {
+        kernel_release => undef,
+        kernel_version => undef,
+        type => undef,
+        analyze => {},
+        blame => {},
+    };
+
+    record_info("BOOT TIME", 'systemd_analyze');
+    # first deployment analysis
+    my ($systemd_analyze, $systemd_blame) = do_systemd_analyze_time($instance);
+    return 0 unless ($systemd_analyze && $systemd_blame);
```

Comment:
Do we really want to ignore `do_systemd_analyze_time` failures?

Reply:
yes. 
1. Not all of them necessarily means something what will block further testing 
2. We do ignoring them at the moment , right ? ( By not actually checking this part on every instance creation ). So there is no change from this POV